.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
+updates:
+  # Enable version updates for npm
+  - package-ecosystem: "npm"
```

Comment:
I assume this refers to the registry, not the package manager, right? If not we may need to change it to Yarn

Reply:
> I assume this refers to the registry, not the package manager.

correct.  it refers to ecosystem, not the manager. package manager is automatically detected.